src/libs/actions/Policy/Category.ts

```diff
@@ -704,7 +704,6 @@ function renamePolicyCategory(policyID: string, policyCategory: {oldName: string
                 value: {
                     [policyCategory.oldName]: null,
                     [policyCategory.newName]: {
-                        ...policyCategoryToUpdate,
                         name: policyCategory.newName,
```

Comment:
In `successData`:

- Remove `[policyCategory.oldName]: null,` as this is done optimistically already
- Remove `name: policyCategory.newName,` as this is done optimistically already
- Move `errors: null,` to the optimistic data as we follow offline first principle and error should be cleared once you queue the request
- Remove `...(policyCategoryToUpdate?.pendingFields ?? {}),` as this is done optimistically already

In `failureData`:
- Remove `name: policyCategory.oldName,` as this is already included in `...policyCategoryToUpdate`
- Remove `pendingAction: null,` as this is already included in `...policyCategoryToUpdate`